testsuite/nestedloop-reg/test.osl

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
```suggestion
// Copyright Contributors to the Open Shading Language project.
// SPDX-License-Identifier: BSD-3-Clause
// https://github.com/AcademySoftwareFoundation/OpenShadingLanguage

```

Reply:
Every year LF runs an automated license scan and sends me a report of all the files that don't clearly reveal licenses. Better to add them from the start than for me to have to go find them months later when I get the report.